metadata.yml

```
@@ -11,11 +16,14 @@ annotations:
   io.artifacthub.displayName: Disallow Service Nodeport
   io.artifacthub.resources: Service
   io.artifacthub.keywords: service
-  # kubewarden specific:
+  # artifacthub specific
   io.kubewarden.policy.ociUrl: ghcr.io/kubewarden/policies/disallow-service-nodeport
   io.kubewarden.policy.author: Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>
   io.kubewarden.policy.title: disallow-service-nodeport
-  io.kubewarden.policy.description: A policy that prevents the creation of Service resources of type `NodePort`
+  io.kubewarden.policy.description: A policy that prevents the creation of Service
+    resources of type `NodePort`
   io.kubewarden.policy.url: https://github.com/kubewarden/disallow-service-nodeport-policy
   io.kubewarden.policy.source: https://github.com/kubewarden/disallow-service-nodeport-policy
   io.kubewarden.policy.license: Apache-2.0
+  io.kubewarden.policy.category: Service
+  io.kubewarden.policy.severity: medium
```

```suggestion
  io.kubewarden.policy.severity: high
```

For routes and ingress, maybe we can use high. Kyverno follows that approach.